deploy/rhsm-clowdapp.yaml

```diff
@@ -12,6 +12,25 @@ parameters:
     value: quay.io/cloudservices/rhsm-subscriptions
   - name: IMAGE_TAG
     value: latest
+  - name: SWATCH_UNLEASH_IMPORT_IMAGE
+    value: quay.io/cloudservices/swatch-unleash-import
+  - name: SWATCH_UNLEASH_IMPORT_IMAGE_TAG
+    value: latest
+  - name: SWATCH_UNLEASH_IMPORT_MEMORY_REQUEST
+    value: 500Mi
+  - name: SWATCH_UNLEASH_IMPORT_MEMORY_LIMIT
+    value: 800Mi
+  - name: SWATCH_UNLEASH_IMPORT_CPU_REQUEST
+    value: 350m
+  - name: SWATCH_UNLEASH_IMPORT_CPU_LIMIT
+    value: 500m
```

Comment:
You can drop these quite a bit I think.  Check out the changes to the curl containers I made in #3740